### PR TITLE
Filter Component

### PR DESCRIPTION
**Pull request by fadeomar**
Relates #44
**Filter Component**
----
-  **Description** 
This component allows users to find the right therapists according to their needs so that the user can choose the appropriate options with him and the component filtering process to show the appropriate results
----
-  **Usage** 
The user must select the appropriate needs for them through the checkboxes and then press the filter button
----
-  **Content** 
Filter Component Component contain : 
    - check-boxes components  
      - Type of therapy.
      - Location.
      - Price range.
    - Remote therapy (toggle button ) .
    - filter button 
    - results component
----
-  **UI** 
Availability Table Component will look like : 
Availability Table Component will

### CODE REVIEW

File: client/src/components/common/Filter/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Select, Form, Button, Switch } from 'antd';
+import { citeies, types, ranges } from './staticData';
```
Comment (Asmaathabet) [inline client/src/components/common/Filter/index.js]: cities ^_^

File: client/src/components/common/Filter/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+class Filter extends Component {
+  handleSubmit = e => {
+    e.preventDefault();
+    const {
+      form: { validateFieldsAndScroll },
+    } = this.props;
+    validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        console.log('Received values of form: ', values);
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Filter/index.js]: There is console.log

Comment (Asmaathabet) [inline client/src/components/common/Filter/index.js]: delete console

File: client/src/components/common/Filter/staticData.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const citeies = [
```
Comment (Asmaathabet) [inline client/src/components/common/Filter/staticData.js]: cities

File: client/src/components/common/Filter/staticData.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  'London',
+  ' Midlands',
```
Comment (MohammadAlhalaq) [inline client/src/components/common/Filter/staticData.js]: space :)

**Pull request merged**